fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -167,11 +156,10 @@ protected void init() {
         return times.values()
                 .stream()
                 .peek(phase -> {
-                    if (phase.start != 0) {
-                        LOG.warn(
-                                "Exporting duration while timer is running. Measurement might be wrong: {}", phase.name
-                        );
+                    if (!phase.isRunning()) {
+                        return;
                     }
+                    LOG.warn("Exporting duration while timer is running. Measurement might be wrong: {}", phase.name);
```

Comment:
This can probably be simplified by using a positive conditional.

```java
if (phase.isRunning()) 
{
   LOG.warn("Exporting duration while timer is running. Measurement might be wrong: {}", phase.name);
}
```